Makefile

```diff
@@ -8,11 +8,16 @@ V8_ARCH = x64
 V8_MODE = release
 
 WASM_FLAGS = -DDEBUG  # -DDEBUG_LOG
-C_FLAGS = ${WASM_FLAGS} -Wall -Werror -ggdb -O -fsanitize=address
-CC_FLAGS = ${C_FLAGS}
-LD_FLAGS = -fsanitize-memory-track-origins -fsanitize-memory-use-after-dtor
+CFLAGS ?= ${WASM_FLAGS} -Wall -Werror -ggdb -O -fsanitize=address
+CXXFLAGS ?= -std=c++11 ${CFLAGS}
+LDFLAGS ?= -fsanitize-memory-track-origins -fsanitize-memory-use-after-dtor
 
-C_COMP = clang
+ifeq ($(origin CC),default)
+CC = clang
+endif
+ifeq ($(origin CXX),default)
+CXX = clang++
```

Comment:
Same here.
Of course, moving this here loses the ability to set both compilers with just one command line var.